apps/web/src/containers/ProjectSelect.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-expect-error FIXME
```

Comment:
I don't like this.
This is a quick and dirty fix for https://github.com/privacy-scaling-explorations/snark-artifacts/actions/runs/10004163116/job/27652343423?pr=124#step:11:14
(I don't really know why this started happening there)